File: Cliente/Cliente_Eventos/Cliente_Eventos/Cliente.Modelo/Cancion.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Cliente_Eventos.Cliente.Modelo
{
    public class Cancion
    {
        public int id { get; set; }
        public string nombre { get; set; }
        public double duracion { get; set; }
        public DateTime fechaLanzamiento { get; set; }
        public string compositor { get; set; }

        public Cancion(int id, string nombre, double duracion, DateTime fechaLanzamiento, string compositor)
        {
            this.id = id;
            this.nombre = nombre;
            this.duracion = duracion;
            this.fechaLanzamiento = fechaLanzamiento;
            this.compositor = compositor;
        }
    }
}

File: Servidor/Servidor_Eventos/src/main/java/com/lausebscode/modelo/Cancion.java
package com.lausebscode.modelo;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.experimental.SuperBuilder;

import java.time.LocalDateTime;

@Data
@NoArgsConstructor
@AllArgsConstructor
@SuperBuilder
public class Cancion {

    private int id;
    private String nombre;
    private double duracion;
    private LocalDateTime fechaLanzamiento;
    private String compositor;


}
